CMakeLists.txt

```diff
@@ -856,6 +856,10 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
 		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/pixmaps)
 	install(FILES Installer/dolphin-emu.desktop
 		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/applications)
+	# Install udev rule
+	install(FILES Installer/51-gcadapter.rules
+		DESTINATION /lib/udev/rules.d
```

Comment:
Actually, it doesn’t work for me on Arch, because pacman thinks that the package wants to override `/lib` (the symlink):

    dolphin-emu-git: /lib exists in filesystem

But it works with `/usr/lib/udev/rules.d`.